vyper/parser/expr.py

```diff
@@ -653,12 +653,6 @@ def load_bytearray(side):
     def parse_BoolOp(self):
         # Iterate through values
         for value in self.expr.values:
-            # Check for calls at assignment
-            if self.context.in_assignment and isinstance(value, vy_ast.Call):
```

Comment:
this was just to handle assert basically?

Reply:
Yeah